_plugins/filters/html_and_meta_util_filter.rb

```diff
@@ -0,0 +1,42 @@
+require_relative '../../lib/utils/meta_util'
+require_relative '../../lib/utils/html_util'
+
+module Filters
```

Comment:
So this is not strictly necessary but you might consider changing the namespace/class name here to something that more closely resembles its place in the file hierarchy... for example:

```ruby
module Filters
  class HtmlAndMetaUtilFilter
    ...
  end
end
```

Reply:
@tcmacdonald Will do that now.